src/explore-education-statistics-admin/src/services/permissionService.ts

```diff
@@ -29,11 +29,6 @@ const permissionService = {
   canUpdateRelease: (releaseId: string): Promise<boolean> => {
     return client.get(`/permissions/release/${releaseId}/update`);
   },
```

Comment:
Can we remove the `canUpdateReleaseFootnotes()` method also now?

Reply:
done